### ISSUE TITLE
Why reexport std types?

### ISSUE THREAD

**Issue by liigo** [myfreeweb/systemstat#74]
data.rs:
```rust
pub use std::time::Duration;
pub use std::net::{Ipv4Addr, Ipv6Addr};
pub use std::collections::BTreeMap;
```

**Comment by myfreeweb** [comment]
yeah, given that the library exports data and doesn't take data from the user, they aren't really necessary

**Comment by dblclik** [comment]
@unrelentingtech - are you looking for a contributor to clean up the `pub use std::...` pieces?  If so, would be happy to help as I plan to use this module in an upcoming project and like to give back when possible

**Comment by valpackett** [comment]
Not really.. ah I should've done this for the 0.2.x version bump. Well, I'll keep this for the next more-interesting API break I guess.
